tutorials/platform/android/index.rst

```diff
@@ -7,3 +7,4 @@ Android plugins
 
    android_plugin
    android_in_app_purchases
+   validation_layers_android
```

Comment:
If we decide to keep it here than the page title would definitely have to be updated. But I'm strongly in favor of moving it under `Engine Development`:
`Engine Development` -> `Vulkan` -> `Validation Layers` -> `Android` (the last one could just be a section in the validation layers page).

Reply:
@m4gr3d Thanks for the comments. Are you ok with creating this structure with empty chapters for now?

Reply:
Yeah that sounds good.

Reply:
Good idea, we'll likely want to add some instructions on how to set up and use validation layers for other platforms indeed. One page for all platforms should likely be fine, starting with just Android for now.

Reply:
I moved the chapter to  `Engine Development` -> `Vulkan` -> `Validation Layers`